src/locales/en/common.json

```diff
@@ -431,6 +431,7 @@
   "Please choose the correct words from the list below to complete your secret recovery phrase.": "Please choose the correct words from the list below to complete your secret recovery phrase.",
   "Please confirm the transaction on your {{deviceModel}}": "Please confirm the transaction on your {{deviceModel}}",
   "Please drag and drop the JSON file from your device.": "Please drag and drop the JSON file from your device.",
+  "Please drop the JSON here": "Please drop the JSON here",
```

Comment:
why this message is required, when we have above `Please drag and drop the JSON file from your device`?

Reply:
It's a message to show when a file is being dragged across the dropzone